aiida_common_workflows/cli/launch.py

```diff
@@ -103,3 +103,102 @@ def cmd_relax(
 
     builder = generator.get_builder(structure, engines, protocol, relaxation_type, threshold_forces, threshold_stress)
     utils.launch_process(builder, daemon)
+
+
+@cmd_launch.command('eos')
+@click.argument('plugin', type=types.LazyChoice(functools.partial(get_workflow_entry_point_names, 'relax', True)))
+@options.STRUCTURE(help='The structure to relax.')
+@options.PROTOCOL(type=click.Choice(['fast', 'moderate', 'precise']), default='fast')
+@options.THRESHOLD_FORCES()
+@options.THRESHOLD_STRESS()
+@options.DAEMON()
```

Comment:
We didn't add for the EOS the new click options `NUMBER_MACHINES` and `WALLCLOCK_SECONDS`. Should we?

Reply:
Yes, good point